<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Publish_ui</class>
 <widget class="QDialog" name="Publish_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#label_picture
{
	border-image: url(:/new/prefix1/m_icon/assignment.png);
}

#frame
{
	background-color: rgb(255, 255, 255);
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>任务名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lETName">
          <property name="placeholderText">
           <string>请输入任务名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_picture">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_language">
          <property name="text">
           <string>任务描述：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="placeholderText">
           <string>示例：该任务内容为翻译莎士比亚的十四行诗.....</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6,1">
        <item>
         <widget class="QLabel" name="label_email">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>翻译类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLanguageForm">
          <item>
           <property name="text">
            <string>汉译英</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>英译汉</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_email_star_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="labelpic_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>翻译文本上传：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETText">
          <property name="placeholderText">
           <string>请上传txt格式文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnText">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 40);</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_email_star_6">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_Proved_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>负责人招募截止日期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditCurrent"/>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEditCurrent"/>
        </item>
        <item>
         <widget class="QLabel" name="label_email_star_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,7,3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 50);
font: 11pt &quot;AcadEref&quot;;</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,7,3">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOK">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 50);
font: 11pt &quot;AcadEref&quot;;</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelpic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>任务封面上传：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETPic">
          <property name="placeholderText">
           <string>请上传png格式文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnPic">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 40);</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,6,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>任务总酬金：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxMoney">
          <property name="maximum">
           <double>99999.990000000005239</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_email_star_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="labelName_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>任务关键词：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lETKeys">
          <property name="placeholderText">
           <string>示例：汉译英、文艺复兴、哥特式文学</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_Proved_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>任务最终完成截止日期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEditFinal">
          <property name="date">
           <date>
            <year>2019</year>
            <month>1</month>
            <day>4</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEditFinal"/>
        </item>
        <item>
         <widget class="QLabel" name="label_email_star_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
